test/e2e/page-objects/pages/token-overview-page.ts

```diff
@@ -0,0 +1,54 @@
+import { Driver } from '../../webdriver/driver';
+
+class TokenOverviewPage {
+  private driver: Driver;
+
+  private readonly sendButton = {
+    text: 'Send',
+    css: '.icon-button',
+  };
+
+  private readonly receiveButton = {
+    text: 'Receive',
+    css: '.icon-button',
+  };
+
+  private readonly swapButton = {
+    text: 'Swap',
+    css: '.icon-button',
+  };
```

Comment:
Using text based selectors here, it's more accessible and means we don't have to create multiple selectors depending on the type of token screen we are looking at. i.e Is the token enabled for swap, bridge, send etc. 

The other selector would be either `coin-overview-*` or `token-overview-*` or `eth-overview-*`